android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt

```diff
@@ -281,40 +283,40 @@ class CalendarDelegate : PluginRegistry.RequestPermissionsResultListener {
             val eventsUri = eventsUriBuilder.build()
             val eventsCalendarQuery = "(${Events.CALENDAR_ID} = $calendarId)"
             val eventsNotDeletedQuery = "(${Events.DELETED} != 1)"
-            val eventsIdsQueryElements = eventIds.map { "(${CalendarContract.Instances.EVENT_ID} = $it)" }
-            val eventsIdsQuery = eventsIdsQueryElements.joinToString(" OR ")
+            val eventsIdsQuery ="(${CalendarContract.Instances.EVENT_ID} IN (${eventIds.joinToString()}))"
 
             var eventsSelectionQuery = "$eventsCalendarQuery AND $eventsNotDeletedQuery"
-            if (eventsIdsQuery.isNotEmpty()) {
+            if (eventIds.isNotEmpty()) {
                 eventsSelectionQuery += " AND ($eventsIdsQuery)"
             }
-            val eventsSortOrder = Events.DTSTART + " ASC"
+            val eventsSortOrder = Events.DTSTART + " DESC"
```

Comment:
Was switching the order of events intentional?

Reply:
A mistake on my part, I'll change it back